avl-auth.cabal

```diff
@@ -66,6 +67,7 @@ test-suite test
     , QuickCheck
     , HUnit
     , quickcheck-instances
+    , xxhash
```

Comment:
this is just a test dependency to "prove" we could derive at least two instances of the `MerkleHash` typeclass.